netjson-monitoring.lua

```diff
@@ -489,7 +503,12 @@ for name, interface in pairs(network_status) do
             end
         end
         if include_stats[name] then
-            netjson_interface.statistics = interface.statistics
+			if is_part_of_bridge(name) then
+				local tx_bytes = interface.statistics.tx_bytes
+				interface.statistics.tx_bytes = interface.statistics.rx_bytes
+				interface.statistics.rx_bytes = tx_bytes
```

Comment:
during testing I realized that there are many different fields beginning with `rx_` and `tx_` (eg: `rx_packets`) that must be inverted, otherwise the data does not make sense.

I think the best way to solve this problem is to create another helper function to call here.

The helper function should:

- loop over all keys of `interface.statistics`
- skip if key does not start with `tx_` 
- from the current `tx_*` key, get the corresponding `rx_*` value and swap the two

Let me know if it's clear.

Reply:
PS: I forgot to mention: let's also add a verbose comment that explains why we're doing this, for future reference